.github/workflows/ci.yml

```
@@ -12,25 +12,25 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10, 11, 12, 13, 14]
+        node-version: [10, 11, 12, 13, 14, 16, 18, 20, 22]
 
     steps:
-    - name: Getting configuration from GitHub
-      uses: actions/checkout@v2
+      - name: Getting configuration from GitHub
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)